### PR TITLE
MOTECH-3015 Added app_id URL parameter handling to FullFormController

### PR DESCRIPTION
**Pull request by mjazdzdewski**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: commcare/src/test/java/org/motechproject/commcare/web/FullFormControllerTest.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ public void testIncomingDeviceReport() throws EndpointNotSupported {
         assertEquals("2012-07-21T15:22:34.046462Z", parameters.get(RECEIVED_ON));
 
         Map<String, String> attributes = (Map<String, String>) parameters.get(ATTRIBUTES);
-        assertEquals(1, attributes.size());
+        assertEquals(2, attributes.size());
```
Comment (kmadej) [inline commcare/src/test/java/org/motechproject/commcare/web/FullFormControllerTest.java:97]: please adjust this test, add app_id value and check if attributes map has this value

File: commcare/src/test/java/org/motechproject/commcare/web/FullFormControllerTest.java [hunk 2]
```diff
@@ -126,7 +126,7 @@ public void testIncomingFormsSuccess() throws EndpointNotSupported {
         assertEquals("2012-07-21T15:22:34.046462Z", parameters.get(RECEIVED_ON));
 
         Map<String, String> attributes = (Map<String, String>) parameters.get(ATTRIBUTES);
-        assertEquals(4, attributes.size());
+        assertEquals(5, attributes.size());
```
Comment (kmadej) [inline commcare/src/test/java/org/motechproject/commcare/web/FullFormControllerTest.java:130]: the same as above

**Review by kmadej — CHANGES_REQUESTED** [review]

**Review by kmadej — APPROVED** [review]

**Pull request merged**